app/models/installed_heat_source.rb

```diff
@@ -0,0 +1,18 @@
+class InstalledHeatSource
+  include Virtus.model
+
+  attribute :key, String
+  attribute :name, String
+  attribute :units, Float
+  attribute :heat_production, Float
```

Comment:
> I am a bit freightened about the capacity term since there's an input_- and output_capacity and @ChaelKruip has a strong opinion about this matter.

Okay! The capacity attribute is not a huge deal; I can work around that and get the heat producers to call `heat_capacity` instead of `capacity`.

Regard `production` though, wouldn't calling it `demand` make it easier to get a demand-scaled network curve? I know that's an InstalledTechnology method, but I'll need to add that to InstalledHeatSource too; maybe that's not a big deal either, since the heat-source version doesn't need to be as complex as the technology one...